mimic.py

```diff
@@ -50,28 +50,36 @@
 def mimic_dict(filename):
     """Returns mimic dict mapping each word to list of words which follow it."""
     with open(filename, "r") as file:
-        text = file.read().split()
-    return map(lambda x:x, text)
+        words = file.read().split()
+        mimic_dict = {}
+    for word in words:
+        prev_word = ''
```

Comment:
Awesome progress! 

Currently, `prev_word` is being reassigned to an empty string each iteration through the for loop. Try assigning `prev_word = ''` above your for loop and reassigning `prev_word` to `word` at the end of your for loop so that the `prev_word` updates each iteration.

Here's an example of the expected data for `mimic_dict`:
`{'': ["Alice's", 'Adventures'], 'ADVENTURES': ['IN', 'WONDERLAND'], 'Adventures': ['in', 'Wonderland'], 'WONDERLAND': ['Lewis'], 'Wonderland': ["ALICE'S", 'ADVENTURES']}`

Here's the an example of what I'm getting for `mimic_dict` from your current code:
`{'': ["Alice's", 'Adventures', 'in', 'Wonderland', "ALICE'S", 'ADVENTURES', 'IN', 'WONDERLAND', 'Lewis', ...]}`